deployments/biology/image/Dockerfile

```diff
@@ -0,0 +1,189 @@
+FROM buildpack-deps:focal-scm
+
+ENV LC_ALL en_US.UTF-8
+ENV LANG en_US.UTF-8
+ENV LANGUAGE en_US.UTF-8
+ENV DEBIAN_FRONTEND=noninteractive
+ENV NB_USER jovyan
+ENV NB_UID 1000
+
+ENV CONDA_DIR /opt/conda
+ENV R_LIBS_USER /opt/r
+
+# Explicitly add littler to PATH
+# See https://github.com/conda-forge/r-littler-feedstock/issues/6
+ENV PATH ${CONDA_DIR}/lib/R/library/littler/bin:${CONDA_DIR}/bin:$PATH
+
+RUN adduser --disabled-password --gecos "Default Jupyter user" ${NB_USER}
+
+# Create user owned R libs dir
+# This lets users temporarily install packages
+RUN mkdir -p ${R_LIBS_USER} && chown ${NB_USER}:${NB_USER} ${R_LIBS_USER}
+
+# Install these without 'recommended' packages to keep image smaller.
+# Useful utils that folks sort of take for granted
+RUN apt-get update -qq --yes && \
+    apt-get install --yes --no-install-recommends -qq \
+        htop \
+        less \
+        man \
+        nano \
+        screen \
+        tar \
+        tmux \
+        wget \
+        vim \
+        locales > /dev/null
+
+RUN echo "${LC_ALL} UTF-8" > /etc/locale.gen && \
+    locale-gen
+
+RUN apt-key adv --keyserver keyserver.ubuntu.com --recv-keys E298A3A825C0D65DFD57CBB651716619E084DAB9
+RUN echo "deb https://cloud.r-project.org/bin/linux/ubuntu focal-cran40/" > /etc/apt/sources.list.d/cran.list
+
+# Install R packages
+# Our pre-built R packages from rspm are built against system libs in focal
+# rstan takes forever to compile from source, and needs libnodejs
+# So we install older (10.x) nodejs from apt rather than newer from conda
+RUN apt-get update -qq --yes > /dev/null && \
+    apt-get install --yes -qq \
+    r-base \
+    r-base-dev \
+    r-recommended \
+    r-cran-littler \
+    nodejs
```

Comment:
We need to add 'npm' here, along with `nodejs`. Otherwise jupyterlab extensions will fail to build